robolectric/src/test/java/org/robolectric/shadows/ShadowWebViewTest.java

```diff
@@ -107,9 +109,7 @@ public void onNewPicture(WebView view, Picture picture) {
       }
     };
 
-    assertThat(shadowWebView.getPictureListener()).isNull();
     webView.setPictureListener(pictureListener);
```

Comment:
If this passes you can just remove the test method entirely. Just wanted you to run it to confirm it doesn't blow up.